src/planscape/planning/views.py

```diff
@@ -51,25 +54,16 @@
 
 # We always need to store multipolygons, so coerce a single polygon to
 # a multigolygon if needed.
-def _convert_polygon_to_multipolygon(geometry: dict):
-    features = geometry.get("features", [])
-    if len(features) > 1 or len(features) == 0:
-        raise ValueError("Must send exactly one feature.")
-    feature = features[0]
-    geom = feature["geometry"]
-    if geom["type"] == "Polygon":
-        geom["type"] = "MultiPolygon"
-        geom["coordinates"] = [feature["geometry"]["coordinates"]]
-
-    actual_geometry = MultiPolygon(
-        [GEOSGeometry(json.dumps(geom)).buffer(0).unary_union]
-    )
-
-    if not actual_geometry.valid:
-        raise ValueError("Geometry is invalid and cannot be used.")
-    if actual_geometry.geom_type != "MultiPolygon":
-        raise ValueError("Could not parse geometry")
-    return actual_geometry
+
+
+@api_view(["POST"])
+def validate_planning_area(request: Request) -> Response:
+    serializer = ValidatePlanningAreaSerializer(data=request.data)
+    serializer.is_valid(raise_exception=True)
+    geometry = coerce_geometry(serializer.validated_data.get("geometry"))
+    data = {"area_acres": get_acreage(geometry)}
+    out_serializer = PlanningAreaValidationSerializer(instance=data)
+    return Response(out_serializer.data)
```

Comment:
I think this looks straightforward enough. 
Is there any value in catching and reraising some of the Exceptions that could happen here as anything besides just a 400 with details?

And do we need auth on this endpoint?